plugin/trino-hudi/src/test/resources/hudi-testing-data/hudi_multi_pt_mor.md

```diff
@@ -0,0 +1,60 @@
+## Create script
+Revision: 6f65998117a2d1228fc96d36053bd0d394499afe
+
+```scala
+test("Create MOR table with multiple partition fields with multiple types") {
+  withTempDir { tmp =>
+    val tableName = "hudi_multi_pt_mor"
+    // Save current session timezone and set to UTC for consistency in test
+    val originalTimeZone = spark.conf.get("spark.sql.session.timeZone")
+    spark.conf.set("spark.sql.session.timeZone", "UTC")
+
+    spark.sql(
+      s"""
+         |CREATE TABLE $tableName (
```

Comment:
For creating testing Hudi tables, should we also use TIMESTAMP and CUSTOM key generators for timestamp partition fields which are also common, for testing partition field values and pruning in Trino Hudi connector?

Reply:
I just checked this. I did not implement the test properly previously. i.e. DynamicFiltering was not truely disabled. This PR includes a proper fix for this so this test is no longer flaky. 

I did it by explicitly setting the system property of DynamicFiltering to false instead of setting the timeout to 0s.

By setting it to 0s previously. DynamicFiltering logic will still be executed and show up in the query analysis, just that the execution will not be given time to finalize the dynamic filters.

By explicitly setting it to false in this fix, it will not perform DynamicFiltering at all and will the regex pattern that i am searching for will not show up in the analysis. Hence, the assertion is changed to `isFalse`.

Reply:
Yeap, i wanted to strike through but it's not really rendering. Not sure why the comment was populated under this comment. 

I added a test to test for TIMESTAMPS, but cannot seem to get the precision working right. The path which the writer wrote to looks weird too and i am not sure if it is buggy or not. I intend to address it later. Will create a Jira for it.

Reply:
I see.